### PR TITLE
create onlythis.md

### PR DESCRIPTION
**Pull request by onlythis**

### CODE REVIEW

**Comment by hikaruit15** [comment]
Hi @onlythis ,
Your page is fine, but it will better if you can add some complex Markdown elements such links or image.

File: index.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
    See http://github.com/Dynalon/mdwiki for a copy of the source code.
 -->
 <head>
-    <title>MDWiki</title>
```
Comment (dogi) [inline index.html]: @onlythis please change this back

**Comment by hannahgdubin** [comment]
Awesome job fixing the index file. I would suggest adding one complex markdown feature. A link or a picture maybe? You can refer to other pages created for ideas.

**Comment by waryhermit** [comment]
@onlythis Do you need any help, or just don't have much time to work on this right now? There are plenty of us who can help you in the [gitter chat](https://gitter.im/open-learning-exchange/chat)

**Comment by waryhermit** [comment]
@onlythis If you don't update this pull request or write in the Gitter chat in 5 days we will assume you are no longer interested in the internship at this time and close the pull request.

**Comment by onlythis** [comment]
Sorry guys, I didn't realize this was happening.

**Comment by dogi** [comment]
@onlythis does this mean you will make an other PR?

**Comment by onlythis** [comment]
@dogi Yes, this weekend.

**Pull request closed without merging**